recipe/meta.yaml

```
@@ -1,29 +1,33 @@
 {% set name = "conda-libmamba-solver" %}
-{% set version = "23.1.0" %}
+{% set version = "23.3.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://github.com/conda/conda-libmamba-solver/archive/{{ version }}.tar.gz  # temporary
```

```suggestion
  url: https://github.com/conda/conda-libmamba-solver/archive/{{ version }}.tar.gz
```

I don't know what was temporary but the current URL doesn't look like it :D